defaults/main.yml

```diff
@@ -0,0 +1,125 @@
+---
+## Package Management
+
+ovos_datadir: "/opt/ovos"
+
+ovos_package_state: "present"
+
+ovos_core_package_names:
+  - "ovos-core"
+  - "ovos-backend-client"
+  - "ovos-audio"
+  - "ovos_plugin_common_play"
+  - "ovos-messagebus"
+  - "ovos-config"
+  - "ovos-utils"
+  - "ovos-bus-client"
+  - "ovos-plugin-manager"
+  - "ovos-workshop"
+  - "ovos-lingua-franca"
+  - "ovos-cli-client"
+
+ovos_phal_plugin_names:
+  - "ovos-PHAL"
+#  - "ovos-phal-plugin-connectivity-events"
+#  - "ovos-phal-plugin-system"
+#  - "ovos-PHAL-plugin-ipgeo"
+#  - "ovos-PHAL-plugin-oauth"
+#  - "ovos-phal-plugin-dashboard"
+#  - "ovos-phal-plugin-alsa"
+#  - "ovos-PHAL-plugin-mk2"
+
+ovos_listener_package_names:
+  - "ovos-dinkum-listener"
+
+ovos_vad_package_name:
+  - "ovos-vad-plugin-silero"
+#  - "ovos-vad-plugin-webrtcvad"
+
+ovos_microphone_package_name:
+  - "ovos-microphone-plugin-alsa"
+
+ovos_ww_package_name:
+  - "ovos-ww-plugin-precise-lite"
+
+ovos_stt_name: 
+  - "ovos-stt-plugin-server"
+
+ovos_tts_name: 
+  - "ovos-tts-plugin-server"
+
+ovos_gui_name: 
+  - "ovos-gui"
+
+ovos_skill_names:
+  - "ovos_skill_date_time"
+  - "ovos_skill_filebrowser"
+  - "ovos_skill_naptime"
+  - "ovos_skill_personal"
+  - "ovos_skill_volume"
+  - "ovos_skill_fallback_unknown"
+  - "ovos_skill_hello_world"
+  - "ovos_skill_stop"
+
+
+## Service Management
+
+ovos_service: "ovos.service"
+ovos_services: 
+  - "ovos-messagebus.service"
+  - "ovos-phal.service"
+  - "ovos-dinkum-listener.service"
+  - "ovos-audio.service"
+  - "ovos-skills.service"
+  - "ovos-admin-phal.service"
+
+## Configuration Management
+
+ovos_lang: "en-en" # it-it
+# metric, imperial
+ovos_system_unit: "metric"
+# half, full 
+ovos_time_format: "half" 
+# MDY, DMY
+ovos_date_format: "DMY"
+ovos_opt_in: "false"
+ovos_confirm_listening: "true"
+# /dev/shm/ovos/cache -> to keep in RAM 
+# /tmp/ovos/cache -> to delete each reboot
+ovos_cache_path: "/tmp/mycroft/cache"  
+# emit mycroft.ready signal when all these conditions are met
+# different setups will have different needs
+# eg, a server does not care about audio
+# pairing -> device is paired
+# internet -> device is connected to the internet - NOT IMPLEMENTED
+# skills -> skills reported ready
+# speech -> stt reported ready
+# audio -> audio playback reported ready
+# gui -> gui websocket reported ready - NOT IMPLEMENTED
+# enclosure -> enclosure/HAL reported ready - NOT IMPLEMENTED
+# network_skills -> skills with network requirements
+# internet_skills -> skills with internet requirements
+ovos_ready_settings: "skills"
+
+ovos_log_path: "/var/log/ovos"
+
+# offline, personal -> https://github.com/OpenVoiceOS/OVOS-local-backend
+ovos_server_backend_type: "offline"
+ovos_server_backend_url: ""
+
+ovos_service_wolfram_key: "{{ secret_wolfram_api_key }}"
+ovos_service_owm_key: "{{ secret_owm_api_key }}"
+
+# STT - ovos-stt-plugin-server
+ovos_stt_plugin_server: "https://stt.openvoiceos.org/stt"
```

Comment:
this should be a list of urls, https://github.com/OpenVoiceOS/ovos-stt-plugin-server/blob/dev/ovos_stt_plugin_server/__init__.py#L11

related https://github.com/OpenVoiceOS/ovos-config/pull/55#issuecomment-1616801044